test/groovy/NpmExecuteEndToEndTestsTest.groovy

```diff
@@ -166,6 +166,114 @@ class NpmExecuteEndToEndTestsTest extends BasePiperTest {
         assert npmExecuteScriptsRule.hasParameter('scriptOptions', ["--baseUrl=http://my-url.com"])
     }
 
+    @Test
+    void oneAppUrlWithWdi5Credential() {
+        def appUrl = [url: "http://my-url.com", credentialId: "testCred"]
```

Comment:
This was tricky: Here it is without 's', i.e. `credentialId`
Later it is with 's', i.e. `credentialsId`